check_graphite_data

```diff
@@ -164,8 +164,8 @@ def main(argv):
 
     if hw:
         graphite_data, graphite_lower, graphite_upper = get_hw_value(url, seconds)
-        print 'Current value: %s, lower band: %s, upper band: %s' % \
-               (graphite_data, graphite_lower, graphite_upper)
+        print 'Current value: %s, lower band: %s, upper band: %s, url: %s'  % \
+               (graphite_data, graphite_lower, graphite_upper, url)
```

Comment:
I'm not currently using nagios, but doesn't the UI allow you to view the command used to run a check, and thus would include the URL?  I feel like this adds visual clutter that isn't necessarily useful for everyone who uses this script, so I'd be more inclined to sign off on the change if the display of the URL were optional.  (Mind you, I don't think I have commit grants on master, so I'm not sure how much my opinion matters here :smile: )